spec/factories/course_forum_topics.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   factory :forum_topic, class: Course::Forum::Topic.name do
-    forum
+    transient do
+      course { build(:course) }
```

Comment:
Actually one of my concern is that the `course_discussion_topic` should not be defined as a factory, it's not supposed to live alone ( lesson plan item is different though ).  But if we have the factory, adding a `parent:` is the best.